cps/__init__.py

```diff
@@ -139,4 +124,34 @@ def get_timezone():
 updater_thread = Updater()
 
 
-__all__ = ['app']
+def main():
+    app = create_app()
+    with app.app_context():
+        from cps.web import web as _web
+        app.register_blueprint(_web)
+
+        from cps.opds import opds as _web_opds
+        app.register_blueprint(_web_opds)
+
+        from cps.jinjia import jinjia as _web_jinjia
+        app.register_blueprint(_web_jinjia)
+
+        from cps.about import about as _web_about
+        app.register_blueprint(_web_about)
+
```

Comment:
This line creates an application error on python3.
```
[2019-08-04 11:43:58,057] DEBUG {cps.uploader:45} cannot import Image, generating pdf covers for pdf uploads will not work: No module named 'wand'
Traceback (most recent call last):
  File "/snap/pycharm-community/143/helpers/pydev/pydevd.py", line 2060, in <module>
    main()
  File "/snap/pycharm-community/143/helpers/pydev/pydevd.py", line 2054, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "/snap/pycharm-community/143/helpers/pydev/pydevd.py", line 1405, in run
    return self._exec(is_module, entry_point_fn, module_name, file, globals, locals)
  File "/snap/pycharm-community/143/helpers/pydev/pydevd.py", line 1412, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/snap/pycharm-community/143/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/.../Entwicklung/calibre-web/cps.py", line 34, in <module>
    _cps_main()
  File "/home/.../calibre-web/cps/__init__.py", line 139, in main
    from cps.about import about as _web_about
  File "/home/.../calibre-web/cps/about.py", line 58, in <module>
    _VERSIONS.update(uploader.get_versions())
  File "/home/matthias/Entwicklung/calibre-web/cps/uploader.py", line 213, in get_versions
    IVersion = _(u'not installed')
  File "/home/.../.local/lib/python3.6/site-packages/flask_babel/__init__.py", line 552, in gettext
    t = get_translations()
  File "/home/.../local/lib/python3.6/site-packages/flask_babel/__init__.py", line 227, in get_translations
    [get_locale()],
  File "/home/.../.local/lib/python3.6/site-packages/flask_babel/__init__.py", line 257, in get_locale
    rv = babel.locale_selector_func()
  File "/home/.../calibre-web/cps/__init__.py", line 103, in negociate_locale
    if request.accept_languages:
  File "/home/...l/lib/python3.6/site-packages/werkzeug/local.py", line 348, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/home/.../.local/lib/python3.6/site-packages/werkzeug/local.py", line 307, in _get_current_object
    return self.__local()
  File "/usr/local/lib/python3.6/dist-packages/flask/globals.py", line 38, in _lookup_req_object
    raise RuntimeError(_request_ctx_err_msg)
RuntimeError: Working outside of request context.

This typically means that you attempted to use functionality that needed
an active HTTP request.  Consult the documentation on testing for
information about how to avoid this problem.
```

Reply:
Forgot to mention "in case Flask-dance" isn't installed (Actually didn't test this on python2,too)

Reply:
I think the exception was actually raised by a missing wand, but indeed the problem was in the about page. Fixed.